### PR TITLE
Add NetMHCpan4 wrapper

### PR DESCRIPTION
**Pull request by timodonnell**

### CODE REVIEW

**Comment by timodonnell** [comment]
Closes #125

File: mhctools/netmhc_pan.py [hunk 1]
```diff
@@ -58,5 +59,13 @@ def NetMHCpan(
...
+    elif "NetMHCpan version 4.0" in output_str:
+        return NetMHCpan4(
+            alleles=alleles,
+            default_peptide_lengths=default_peptide_lengths,
+            program_name=program_name,
+            process_limit=process_limit,
+            extra_flags=extra_flags)
+
     else:
         raise SystemError("This software expects NetMHCpan version 2.8 or 3.0")
```
Comment (tavinathanson) [inline mhctools/netmhc_pan.py]: Fix this message?

File: mhctools/netmhc_pan.py [hunk 2]
```diff
@@ -58,5 +59,13 @@ def NetMHCpan(
             process_limit=process_limit,
             extra_flags=extra_flags)
 
+    elif "NetMHCpan version 4.0" in output_str:
```
Comment (tavinathanson) [inline mhctools/netmhc_pan.py]: Nit, but maybe merge this and the other 2 sections into:

```
version_class = None
if "NetMHCpan version 2.8" in output_str:
    version_class = NetMHCpan28
elif "NetMHCpan version 3.0" in output_str:
    version_class = NetMHCpan3
...
else:
    raise ...

return version_class(...
```

File: test/test_netmhc_pan.py [hunk 3]
```diff
@@ -10,8 +10,23 @@
     "TP53-001": "ASILLLVFYW"
 }
 
+# This test assumes you have versions of netMHCpan installed in your PATH
```
Comment (tavinathanson) [inline test/test_netmhc_pan.py]: Not sure we should assume that people running `mhctools` tests have all versions installed. Maybe make this optional but have it run on Travis?

**Review by tavinathanson — APPROVED** [review]
LGTM % comments

File: mhctools/netmhc_pan.py (hunk 1 above)
Comment (timodonnell) [reply mhctools/netmhc_pan.py]: Good catch, fixed

File: mhctools/netmhc_pan.py (hunk 2 above)
Comment (timodonnell) [reply mhctools/netmhc_pan.py]: Changed it to something similar but a little less clever

File: test/test_netmhc_pan.py (hunk 3 above)
Comment (timodonnell) [reply test/test_netmhc_pan.py]: Agree, updated to make the versioned names optional. The test will run those versions if they exist

**Comment by timodonnell** [comment]
Thanks for the review, updated to address comments

File: mhctools/netmhc_pan.py (hunk 2 above)
Comment (mhcbundle) [reply mhctools/netmhc_pan.py]: Cool, even better

Comment (mhcbundle) [reply mhctools/netmhc_pan.py]: Whoops, I've revealed my secret identity

**Pull request merged**